src/interfaces/assistants_web/src/components/Conversation/Composer/WebSearchConfigurationModal.tsx

```diff
@@ -18,13 +17,6 @@ export const WebSearchModal: React.FC<{
     params: { tools },
   } = useParamsStore();
   const [site, setSite] = useState('');
-  const inputRef = useRef<HTMLInputElement>(null);
-
-  useEffect(() => {
-    if (inputRef.current) {
-      inputRef.current.focus();
-    }
-  }, []);
```

Comment:
I think we should keep this

Reply:
this component is not used and is not part of the new designs, will most likely be deleted